docs/types.rst

```diff
@@ -914,6 +921,54 @@ Of course, you can also directly access the members of the struct without
 assigning it to a local variable, as in
 ``campaigns[campaignID].amount = 0``.
 
+
+Types as input and output parameters
+====================================
+
+.. index:: !function parameters, functions, parameters
+
+Allowed function parameter types
+--------------------------------
+
+**All** :ref:`value-types` variables are allowed as input and output function parameter types.
```

Comment:
Does this render as "value types variables"? If yes, it sounds rather weird to me.